etna/models/autoarima.py

```diff
@@ -56,6 +56,8 @@ class AutoARIMAModel(
     """
     Class for holding auto arima model.
 
+    This is autoregression model, take this into account using ``forecast`` and ``predict`` methods.
```

Comment:
Yes, we just inherit the implementation of `forecast`/`predict` from the mixins.

But behavior of this these mixins is different depending on the methods that implemented in adapters.

I expect if adapter implements only `predict` then it is regression model, if it implements both then it is auto-regression model.

I just think that we should have some way to tell the user how this particular model will behave: can it make use of true values on train data or not. Do you have any ideas how to make this information visible and checkable?

Reply:
```python
class AutoARIMAModel(
    PerSegmentModelMixin, PredictionIntervalContextIgnorantModelMixin, PredictionIntervalContextIgnorantAbstractModel
)
```

It seems this explain everything, isn't it?

Reply:
How? It doesn't tell you how this model will make `forecast`: call `predict` or use special method implemented in adapter for dealing with in-sample.

Currently we have in `forecast`:
```
        * If it is regression model, the results of ``forecast`` and ``predict`` are the same.

        * If it is autoregression model, this method will make autoregressive predictions.

        To understand how a particular model behaves look at its documentation.
```

In `predict`:
```
        * If it is regression model, the results of ``forecast`` and ``predict`` are the same.

        * If it is autoregression model, this method will make predictions using true values
          instead of predicted on a previous step.
          It can be useful for making in-sample forecasts.

        To understand how a particular model behaves look at its documentation.
```